installing/installing_aws/installing-restricted-networks-aws.adoc

```diff
@@ -150,5 +150,6 @@ include::modules/installation-aws-user-infra-installation.adoc[leveloffset=+1]
 == Next steps
 
 * xref:../../installing/install_config/customizations.adoc#customizations[Customize your cluster].
+* If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by xref:../../openshift_images/image-configuration.adoc#images-configuration-cas_image-configuration[Configuring additional trust stores].
```

Comment:
Since you're not explicitly referring to the link as an article, but rather an action, I would recommend lower-casing the link title:
> If the mirror registry that you used to install your cluster has a trusted CA, add it to the cluster by [configuring additional trust stores]().

Same goes for the other links in this PR.